tests/onnx/test_e2e_ptq.py

```diff
@@ -330,32 +346,96 @@ def get_ac_command(task_type: str, model_name: str, model_dir: Path,
-        command = self.get_ac_command(task_type, model_name, model_dir, data_dir, anno_dir, output_dir, eval_size,
-                                      program="accuracy_checker.py", is_quantized=False, is_ov_ep=False, is_cpu_ep=True)
+        command = self.get_onnx_rt_ac_command(task_type, model_name, model_dir, data_dir, anno_dir, output_dir,
+                                              eval_size, program="accuracy_checker.py", is_quantized=False,
+                                              is_ov_ep=False, is_cpu_ep=True)
         run_command(command)
 
     @pytest.mark.e2e_ptq
     @pytest.mark.dependency()
-    @pytest.mark.parametrize("task_type, model_name", MODELS)
-    def test_quantized_model_accuracy(self, request, task_type, model_name, model_names_to_test, ckpt_dir, data_dir,
-                                      anno_dir, output_dir, eval_size, is_ov_ep, is_cpu_ep):
+    @pytest.mark.parametrize("task_type, model_name", E2E_MODELS)
+    def test_onnx_rt_quantized_model_accuracy(self, request, task_type, model_name, model_names_to_test, ckpt_dir,
+                                              data_dir, anno_dir, output_dir, eval_size, is_ov_ep, is_cpu_ep):
+        if not (is_ov_ep or is_cpu_ep):
+            pytest.skip('Skip accuracy validation on ONNXRuntime.')
         check_skip_model(model_name, model_names_to_test)
         # Run PTQ first
-        depends(request, ["TestPTQ::test_ptq_model" + request.node.name.lstrip("test_quantized_model_accuracy")])
+        depends(request,
+                ["TestPTQ::test_ptq_model" + request.node.name.lstrip("test_onnx_rt_quantized_model_accuracy")])
         check_xfail(model_name)
         check_quantized_xfail(model_name)
 
         model_dir = ckpt_dir
-        command = self.get_ac_command(task_type, model_name, model_dir, data_dir, anno_dir, output_dir, eval_size,
-                                      program="accuracy_checker.py", is_quantized=True, is_ov_ep=is_ov_ep,
-                                      is_cpu_ep=is_cpu_ep)
+        command = self.get_onnx_rt_ac_command(task_type, model_name, model_dir, data_dir, anno_dir, output_dir,
+                                              eval_size, program="accuracy_checker.py", is_quantized=True,
+                                              is_ov_ep=is_ov_ep, is_cpu_ep=is_cpu_ep)
+        run_command(command)
+
+    @pytest.mark.e2e_ptq
+    @pytest.mark.dependency()
+    @pytest.mark.parametrize("task_type, model_name", E2E_MODELS)
+    def test_ov_quantized_model_accuracy(self, request, task_type, model_name, model_names_to_test, ckpt_dir, data_dir,
+                                         anno_dir, output_dir, eval_size, is_ov):
+        if not is_ov:
+            pytest.skip('Skip accuracy validation on OpenVINO.')
+        check_skip_model(model_name, model_names_to_test)
+        # Run PTQ first
+        depends(request, ["TestPTQ::test_ptq_model" + request.node.name.lstrip("test_ov_quantized_model_accuracy")])
```

Comment:
You are trying to use `.lstrip` to remove a prefix, which is wrong because it does not actually do what you want it to do. Use some of the suggestions in https://stackoverflow.com/questions/16891340/remove-a-prefix-from-a-string instead.

Reply:
Done